### PR TITLE
Create samples with packet energies

### PR DESCRIPTION
**Pull request by Knights-Templars**
### :pencil: Description

**Type:** :rocket: `feature` | :roller_coaster: `infrastructure`

Create an array with packet energies sampled randomly from the decay data frame.

### :vertical_traffic_light: Testing

How did you test these changes?

- [X] Testing coming soon.

### :ballot_box_with_check: Checklist

- [X] I requested two reviewers for this pull request

### CODE REVIEW

File: tardis/energy_input/gamma_ray_channel.py [hunk 1]
```diff
@@ -167,3 +167,80 @@ def create_isotope_decay_df(cumulative_decay_df, gamma_ray_lines):
...
+
+    Returns
+    -------
+    sampled_packet : pd.Series
+        A Series of sampled packets using the normalized energy in each channel.
+
+    """
+
+    sampled_packet = isotope_decay_df.sample(
+        weights="normalized_energy", random_state=np.random.RandomState(seed=29)
```
Comment (andrewfullard) [inline tardis/energy_input/gamma_ray_channel.py:215]: pass seed as function argument

File: tardis/energy_input/gamma_ray_channel.py [hunk 2]
```diff
@@ -167,3 +167,80 @@ def create_isotope_decay_df(cumulative_decay_df, gamma_ray_lines):
     )
 
     return isotope_decay_df
+
+
+def distribute_packets(isotope_decay_df):
```
Comment (DeerWhale) [inline tardis/energy_input/gamma_ray_channel.py:172]: pandas dataframe.sample function do the normalized weights sampling, so maybe this function is not needed to save computational resource.

Comment (Knights-Templars) [reply tardis/energy_input/gamma_ray_channel.py:172]: Resolved!

File: tardis/energy_input/gamma_ray_channel.py (hunk 1 above)
Comment (Knights-Templars) [reply tardis/energy_input/gamma_ray_channel.py:215]: done!

**Comment by Knights-Templars** [comment]
Sampling will be made in the PacketSource.

**Pull request closed without merging**